Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine AS build
+RUN apk update && apk add \
+    musl-dev \
```

Comment:
Did you choose musl to shrink the size of the image or because ptunnel-ng gets already built against musl for OpenWrt? I am just curious.

Reply:
Like OpenWrt, `musl` is the default C library for Alpine Linux.